system/Database/OCI8/Connection.php

```diff
@@ -384,40 +384,59 @@ protected function _indexData(string $table): array
     protected function _foreignKeyData(string $table): array
     {
         $sql = 'SELECT
```

Comment:
This query had lots of problems. First I wanted to get the delete rule so I changed some things there. Then the other problem is that the query was improperly joined and was returning duplicate rows. The additional join condition: accu.POSITION = acc.POSITION fixed this problem.